package.json

```diff
@@ -19,11 +19,11 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://finrem-cos-aat.service.core-compute-aat.internal",
+      "cosUrl": "http://finrem-cos-pr-571.service.core-compute-preview.internal",
```

Comment:
these will need to be merged so build will go green (new endpoints only exist in preview), these will be reverted after COS merge